modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/index.es.js

```diff
@@ -26,6 +26,10 @@ export {default as autoSize} from './liferay/autosize/autosize.es';
 
 export {cancelDebounce, debounce} from './liferay/debounce/debounce.es';
 
+// Delegate API
+
+export {delegate} from './liferay/delegate/delegate.es';
```

Comment:
I think this shouldn't work now, since you're importing named but exporting default... 🤔

Reply:
That makes sense, but I tested it and it worked, maybe some caching was going on, even though I noticed changes were being made. Interesting, I'll check it out.